generate_test_data/README.md

```diff
@@ -0,0 +1,9 @@
+# test_webapp_data_gen
+
+Generate test data for web app. Place `pcpt_hourly_wrf_ERA-Interim_historical_1992.nc` in this folder, run `generate_data.py`. 
```

Comment:
It doesn't looks like there's a `Pipenv` for this?  I'd suggest adding one!  Approximately:

```
pipenv run python generate_data.py
```

(pipenv will do a bunch of things and create files and stuff.  Commit the `Pipfile` and `Pipfile.lock` to the repo as part of the commit).

Keep running that command and doing `pip install whatever` until it works!

Commit & update docs!  Done!